pages/index.tsx

```diff
@@ -264,8 +265,8 @@ for await (const req of s) {
               >
                 <h4 className="text-lg font-bold">{s.name}</h4>
                 <p
-                  className="whitespace-normal break-words text-gray-700 mt-2"
-                  style={{ textOverflow: "ellipsis" }}
+                  className="whitespace-normal break-words text-gray-700 mt-2 overflow-hidden"
+                  style={{ textOverflow: "ellipsis", minHeight: "4.5em", WebkitLineClamp: 3, WebkitBoxOrient: "vertical", display: "-webkit-box" }}
```

Comment:
I'm having trouble with prettier not wanting to format, hence the linter gives an error.

Reply:
Did you run `yarn format`?

Reply:
I ran it by hand, because I wanted to avoid installing yarn just for this (I use npm :) ) I now noticed I missed the write flag. It writes all the files, but I can stage only the one I modified so this is sorted now